sky/engine/core/dart/window.dart

```diff
@@ -91,6 +91,10 @@ class Window {
   /// A callback that is invoked whenever [locale] changes value.
   VoidCallback onLocaleChanged;
 
+  /// Callbacks invoked when the activity is paused or resumed.
```

Comment:
This comment will apply to onActivityPaused and onActivityResumed will have no comment. You should describe each separately with its own comment.